e2e/go.mod

```diff
@@ -214,4 +214,4 @@ replace (
 )
 
 // uncomment to use the local version of ibc-go, you will need to run `go mod tidy` in e2e directory.
-// replace github.com/cosmos/ibc-go/v6 => ../
+replace github.com/cosmos/ibc-go/v6 => ../
```

Comment:
should we recomment this?

Reply:
we want this commented out as this comment is simply here for convenience to allow for testing against local ibc-go changes.